dev/VSIX/Directory.Build.targets

```diff
@@ -1,12 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
something to consider: one thing I didn't get around to when porting from winui, was to remove the need to revert direct edits to source files by first copying them to a staging folder, as cppwinrt does:
[https://github.com/microsoft/cppwinrt/blob/342edc5290086334366881faf8c4564bba00625b/vsix/vsix.csproj#L153](https://github.com/microsoft/cppwinrt/blob/342edc5290086334366881faf8c4564bba00625b/vsix/vsix.csproj#L153)

Reply:
Yes, that's an excellent suggestion; it'd likely also resolve the issue I've run into sometimes wherein VS decides it doesn't need to run the revert target, forcing me to manually rebuild. I'll make a note to implement this next time I'm in this space.